client/app/bundles/course/user-email-subscriptions/operations.ts

```diff
@@ -1,30 +1,37 @@
-import CourseAPI from 'api/course';
-
-import actionTypes from '../constants';
+import { Operation } from 'store';
 
-import { setNotification } from './index';
+import CourseAPI from 'api/course';
+import { setNotification } from 'lib/actions';
 
-export async function fetchUserEmailSubscriptions(params) {
-  const response = await CourseAPI.userEmailSubscriptions.fetch(params);
+import actionTypes from './constants';
 
-  return response.data;
+export function loadUserEmailSubscriptions(data): Operation {
```

Comment:
Do we need to export this?

Reply:
No, we don't. At first I was exporting this because I was using it for the tests (I haven't had `setUpTestStore` back then).